src/lemlib/odom/differentialArc.cpp

```diff
@@ -32,33 +32,33 @@ void DifferentialArc::calibrate(bool calibrateGyros) {
     // calibrate vertical tracking wheels
     for (auto it = verticals.begin(); it != verticals.end(); it++) {
         if (it->reset()) {
-            infoSink()->warn("Vertical tracker at offset {} failed calibration!", it->getOffset());
+            infoSink()->warn("Vertical tracker at offset {} failed calibration!", it->getOffset().convert(in));
             verticals.erase(it);
         }
     }
 
     // calibrate horizontal tracking wheels
     for (auto it = horizontals.begin(); it != horizontals.end(); it++) {
         if (it->reset()) {
-            infoSink()->warn("Horizontal tracker at offset {} failed calibration!", it->getOffset());
+            infoSink()->warn("Horizontal tracker at offset {} failed calibration!", it->getOffset().convert(in));
             horizontals.erase(it);
         }
     }
 
     // calibrate drivetrain motors
     for (auto it = drivetrain.begin(); it != drivetrain.end(); it++) {
         if (it->reset()) {
-            if (sgn(it->getOffset() == 1))
-                infoSink()->warn("Left drivetrain motor failed to calibrate!", it->getOffset());
-            else infoSink()->warn("Right drivetrain motor failed to calibrate!", it->getOffset());
+            if (units::sgn(it->getOffset()) == 1)
+                infoSink()->warn("Left drivetrain motor failed to calibrate!", it->getOffset().convert(in));
+            else infoSink()->warn("Right drivetrain motor failed to calibrate!", it->getOffset().convert(in));
             drivetrain.erase(it);
         }
     }
 
     if (!calibrateGyros) return; // return if we don't need to calibrate gyros
     // calibrate gyros
     for (auto& it : gyros) it->calibrate();
-    Timer timer(3000); // try calibrating gyros for 3000 ms
+    Timer timer(3_sec); // try calibrating gyros for 3000 ms
```

Comment:
```suggestion
    Timer timer(3_sec); // try calibrating gyros for 3 seconds
```
Update comment to use the right unit